experimenter/experimenter/experiments/api/v8/urls.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework.routers import SimpleRouter
+
+from experimenter.experiments.api.v8.views import (
+    NimbusExperimentDraftViewSet,
+    NimbusExperimentFirstRunViewSet,
+    NimbusExperimentViewSet,
+)
+
+router = SimpleRouter()
+router.register(r"experiments", NimbusExperimentViewSet, "nimbus-experiment-rest-v8")
+router.register(
+    r"experiments-first-run",
+    NimbusExperimentFirstRunViewSet,
+    "nimbus-experiment-rest-v8-first-run",
+)
```

Comment:
I think we can safely omit this from v8, first-run isn't used by jetstream as far as I know.
```suggestion
```